app/Http/Controllers/SupportController.php

```diff
@@ -252,8 +252,13 @@ public function removeSupport(Request $request)
 
         try{
 
-                TopicSupport::removeDirectSupport($topicNum, $removeCamps, $nickNameId, $action, $type, $orderUpdate, $request->user());                
-               // PushNotification::pushNotificationToSupporter($request->user(),$topicNum, $campNum, 'remove');
+                TopicSupport::removeDirectSupport($topicNum, $removeCamps, $nickNameId, $action, $type, $orderUpdate, $request->user());     
+                // $nickName = '';
```

Comment:
We can remove the commented code if not required

Reply:
Done